tests/app/count.js

```diff
@@ -26,27 +27,31 @@ describe('counter', function () {
     console.log = function (val) {
       nums.push(val);
     };
+
+    this.clock = sinon.useFakeTimers();
   });
 
   afterEach(function () {
     console.log = origConsoleLog;
+
+    this.clock.restore();
   });
 
   it('should count from start number to end number, one per 1/10th of a second', function (done) {
     this.timeout(600);
     countAnswers.count(1, 5);
 
-    setTimeout(function () {
-      expect(nums.length > 1).to.be.ok;
-      expect(nums.length < 5).to.be.ok;
-    }, 200);
-
-    setTimeout(function () {
-      expect(nums.length).to.eql(5);
-      expect(nums[0]).to.eql(1);
-      expect(nums[4]).to.eql(5);
-      done();
-    }, 550);
+    for (var i = 1; i <= 5; i++) {
+      expect(nums.length).to.eql(i);
+
+      this.clock.tick(100);
+    }
+
+    expect(nums.length).to.eql(5);
+    expect(nums[0]).to.eql(1);
+    expect(nums[4]).to.eql(5);
+
+    done();
```

Comment:
The tests might (should?) be synchronous now, so we may be able to get rid of this `done()` call.